portfolio/src/main/java/com/google/sps/servlets/GetCommentsServlet.java

```diff
@@ -31,15 +31,15 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/comments")
-public class DataServlet extends HttpServlet {
+/** Servlet that returns comments data */
+@WebServlet("/get-comments")
+public class GetCommentsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int maxComments = getMaxComments(request);
 
-    Query query = new Query("Comment").addSort("postTime", SortDirection.ASCENDING);
+    Query query = new Query("Comment").addSort("postTime", SortDirection.DESCENDING);
```

Comment:
I suggest extracting things like "Comment", "text", and "postTime" out to their own constants.